week1/homework/src/server.js

```diff
@@ -10,10 +10,35 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    const requestUrl = request.url;
+    switch (requestUrl) {
+      case '/state':
+      response.statusCode = 200;
+      response.setHeader('content-Type', 'application/json');
```

Comment:
This line can be moved to before the switch block, because it is repeated in all cases.